sdk/physical/cache.go

```diff
@@ -412,3 +412,11 @@ func (c *cacheTransaction) Rollback(ctx context.Context) error {
 
 	return nil
 }
+
+func (c *cache) Invalidate(ctx context.Context, key string) {
```

Comment:
Do you mind just adding a note saying that it is expected not to invalidate keys in the transactions here? The cache for a transaction is empty at the time the transaction started and any in-flight transaction would not see any new storage updates so would not need to be invalidated either way.

Reply:
I think it wouldn't hurt.

Reply:
done